tools/kmodules/sched_tp/sched_tp.c

```diff
@@ -0,0 +1,132 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+#include <linux/module.h>
+
+#include <linux/sched.h>
+#include <trace/events/sched.h>
+
+#define CREATE_TRACE_POINTS
+#include "sched_events.h"
+
+static inline struct cfs_rq *get_group_cfs_rq(struct sched_entity *se)
+{
+#ifdef CONFIG_FAIR_GROUP_SCHED
+       return se->my_q;
+#else
+       return NULL;
+#endif
+}
+
+static void sched_pelt_cfs(void *data, struct cfs_rq *cfs_rq)
+{
+	if (trace_sched_load_cfs_rq_enabled()) {
```

Comment:
I don't mind but I kept it like that so one can insert their own code around this more easily if they too (in case anyone felt extending what they want to do when this tracepoint is triggered)